src/PasswordResetController.js

```diff
@@ -45,6 +45,32 @@ function (Okta, FormController, FormType, ValidationUtil, FooterSignout, TextBox
     Form: {
       save: Okta.loc('password.reset', 'login'),
       title: Okta.loc('password.reset.title', 'login'),
+      subtitle: function () {
+        var policy = this.options.appState.get('policy');
+        if (policy.complexity) {
```

Comment:
Also, are we always guaranteed to have policy? I.e. if you don't have some feature flag/etc, is policy always defined or will this line throw an error?